orm/config-file.md

```diff
@@ -0,0 +1,316 @@
+    |
+    | Configure meta-data, query and result caching here.
+    | Optionally you can enable second level caching.
+    |
+    | Available: acp|array|file|memcached|redis
+    |
+    */
+    'cache'                     => [
+        'default'                => env('DOCTRINE_CACHE', 'array'),
+        'namespace'              => null,
+        'second_level'           => false,
+    ],
+    /*
+    |--------------------------------------------------------------------------
+    | Gedmo extensions
+    |--------------------------------------------------------------------------
+    |
+    | Settings for Gedmo extensions
+    | If you want to use this you will have to require
+    | laravel-doctrine/extensions in your composer.json
+    |
+    */
+    'gedmo'                     => [
+        'all_mappings' => false
+    ]
+];
+```
+
+### Entity Manager
+
+An **Entity Manager (EM)** contains all of the information Doctrine needs to understand, retrieve, and manipulate a set of entities (models). 
+
+You must have **at least one** EM in order to use Laravel Doctrine.
+
+To use more than one EM simply create another entry in the `managers` array.
+
+* **EM Name** - In the sample below the EM we have configured is named `default`. This is the EM that Laravel Doctrine will attempt to use if no argument is provided to `ManagerRegistry`.
+* **dev** - Whether this EM is in development mode.
+* **meta** - The metadata driver to use. Built-in options are `annotations|yaml|xml|config|static_php`
+* **connection** - The connection drier to use. Built-in options are `mysql|oracle|pgsql|sqlite|sqlsrv`
+* **namespaces** - (Optional) If your entities are not located in the configured app namespace you can specify a different one here.
+* **paths** - An paths where the mapping configurations for your entities is located.
+* **repository** - (Optional) The default repository to use for this EM.
+* **proxies** - (Optional) Configuration settings for proxy classes for your entities.
+ * **namespace** - Namespace (if different) specified for proxy classes
+ * **path** - The path where proxy classes should be generated.
+ * **auto_generate** - Should proxy classes be generated every time an EM is created? (Turn off production)
+
+
+```php
+    'managers'                  => [
+        'default' => [
+            'dev'        => env('APP_DEBUG'),
+            'meta'       => env('DOCTRINE_METADATA', 'annotations'),
+            'connection' => env('DB_CONNECTION', 'mysql'),
+            'namespaces' => [
+                'App'
+            ],
+            'paths'      => [
+                base_path('app')
+            ],
+            'repository' => Doctrine\ORM\EntityRepository::class,
+            'proxies'    => [
+                'namespace'     => false,
+                'path'          => storage_path('proxies'),
+                'auto_generate' => env('DOCTRINE_PROXY_AUTOGENERATE', false)
+            ],
+            'events' => ...
```

Comment:
I left the "Events and Filters" section empty because I don't use it much and don't know much about it. @josenicomaia or @patrickbrouwers could you do this?